doc/en/configuration_guide/poller/02configuration.rst

```diff
@@ -15,29 +15,58 @@ Once the installation is completed, it is necessary to integrate this remote ser
    :align: center
 
 
-#. Go into the menu: **Configuration** ==> **Pollers** ==> **Engine configuration**
+#. Go into the **Configuration > Pollers > Engine configuration** menu
 #. Select your last added configuration.
 #. Change the following settings, and save:
 
-*	In the **Data** tab - **Multiple broker module** field change the name of the of Centreon Broker configuration file **central-module.xml** to for example: poller1-module.xml.
+* In the **Files** tab:
+
+  * Modify **Configuration Name**
+  * Check that ** Linked poller** is the previous created poller
+  * Change if necessary the **Timezone / Location**
+
+.. image:: /images/user/configuration/10advanced_configuration/07addengine.png
+   :align: center
+
+* In the **Data** tab - **Multiple Broker Module** fields check / add the following entries::
+
+   /usr/lib64/centreon-engine/externalcmd.so
```

Comment:
> /usr/lib64/centreon-engine/externalcmd.so

this entry conflits with the following print: 

https://raw.githubusercontent.com/centreon/centreon/fe43b804536080268f51a0ab911c7c6657ca5160/doc/en/images/user/configuration/10advanced_configuration/07addpoller_neb.png